espree.js

```diff
@@ -5114,7 +5113,9 @@ function parseClassDeclaration() {
 
     expectKeyword("class");
 
-    id = parseVariableIdentifier();
+    if (lookahead.type === Token.Identifier) {
```

Comment:
Let me know if you have a specific `class` declaration that breaks, at the moment I think this code-path may only be called in `export` statements, so maybe it works by accident? maybe.